### ISSUE TITLE
U-Net is not converging in RTX3090

### ISSUE THREAD

**Issue by yilei-wu** [tensorflow/tensorflow#50021]
<em>Please make sure that this is an issue related to performance of TensorFlow.
As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:performance_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): CentOS 7.9
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: NA
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 1.14.0(both tensorflow and tensorflow-gpu)
- Python version: 2.7
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: CUDA-11.0, cudatoolkit-10.0, cudnn-7.6.5
- GPU model and memory: GTX3090(24G mem)

**Describe the current behavior**
packages(tensorflow, cudnn, cudatoolkit) installed using anaconda; 
Tyring to replicate result from a U-Net project(https://github.com/hongweilibran/wmh_ibbmTum) by running train_leave_one_out.py; 
The model cannot converge, loss not drop from the beginning. Same code worked as expected on Google colab and other remote runtime environment.

Other TF2.X and pytorch toy project works fine on this runtime.
Any input is appreaciated!

**Comment by UsharaniPagadala** [comment]
@yilei-wu 

I tried running the code on colab didn't face any errors as you mentioned,here is the [gist](https://colab.research.google.com/gist/UsharaniPagadala/357a13728f737da6b337be8f87ddedca/untitled76.ipynb) as you mentioned
Could you please check the Tensorflow and cuda and cuDNN versions  compatibility of your env as the latest tested build configurations are these.

Version | Python version | Compiler | Build tools | cuDNN | CUDA
-- | -- | -- | -- | -- | --
tensorflow-2.5.0 | 3.6-3.9 | GCC 7.3.1 | Bazel 3.7.2 | 8.1 | 11.2
tensorflow-2.4.0 | 3.6-3.8 | GCC 7.3.1 | Bazel 3.1.0 | 8.0 | 11.0

 please upgrade to tf2.x  as we don't support for tf1.x,let me know if It helps
For Anaconda env related issues  post [here](https://github.com/ContinuumIO/anaconda-issues/issues) for better response.
Thanks

**Comment by yilei-wu** [comment]
Thank you for your reponse!

Yes, I did not observe errors on the google colab as well and the colab dependency is not exactly same as it used in the original repo.

The project is written in TF1.X therefore I have to rewirte the some part to make it runnable on TF2.X.
Another problem is I cannot choose the offcial-compatible CUDA version since that is not installed on the HPC(the problematic runtime).

My question is is there possible explanations casuing the model not converge using TF1.14 with GTX3090 and corresponding cuda? Based on your knowledge, would the compatbility issue cause this problem?

**Comment by UsharaniPagadala** [comment]
@yilei-wu 
Could you please refer [this](https://github.com/tensorflow/tensorflow/issues/44200),it helps.Thanks
